src/solvers/petscdmlibmeshimpl.C

```diff
@@ -1203,7 +1203,7 @@ PetscErrorCode  DMCreate_libMesh(DM dm)
   dm->ops->getinjection       = 0; // DMGetInjection_libMesh;
   dm->ops->getaggregates      = 0; // DMGetAggregates_libMesh;
 #else
-  dm->ops->createinjection = 0;
+  //dm->ops->createinjection = 0;
```

Comment:
What's going on here?

Reply:
Accidentally  changed this when I was trying different versions of PETSc.

I will change it back .

Reply:
:+1: IMHO after this change is reverted.

Reply:
Ping?